### DISCUSSION TITLE
nav_card colors

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1440] [Ideas]
Proposal to add another `color` parameter option to `nav_card()`: `transparent`, just like `header_card()`.

This will allow making side bar navigation have a consistent look to H2O AI Cloud App Store.

![image](https://user-images.githubusercontent.com/64787868/170131261-62a4733c-fa21-425c-96f4-28a9c2c0c3ae.png)

**Comment by mturoci** [comment]
This sounds reasonable, can you please file a feature request @azim-b? Thanks!
